Gemfile

```diff
@@ -59,6 +59,7 @@ group :development, :test do
   gem "parallel_tests"
   gem "pry-byebug"
   gem "pry-rails"
+  gem "rspec-default_http_header"
```

Comment:
I was wondering if we really need this dependency, and how complex it would be to set up the authorisation using normal requests or a helper?

Reply:
Without this gem we'll have to pass in `headers` to every `get`, `post` and `put`.
eg:
```ruby
get("/api/v3/statements", headers: { "Authorization" => "Bearer #{token}" })
```